### PR TITLE
Changelog 1.0.5.6

### PR DESCRIPTION
**Pull request by NicolasDorier**
Still want to add #1908 and #1841 before release

### CODE REVIEW

**Review by Kukks — CHANGES_REQUESTED** [review]

**Comment by NicolasDorier** [comment]
Should we include https://github.com/btcpayserver/btcpayserver/pull/1930 ?

**Comment by dennisreimann** [comment]
@NicolasDorier It's not fully implemented yet (pull payments UI missing) and given the variety of different states we should test, I think we should not rush it.

**Comment by pavlenex** [comment]
I think it's best we leave out the #1930, as the time-frame is short and as Dennis mentioned we need a bit more time to test it out, so best not to rush it (we have plenty of interesting stuff ready for the release on Thursday already).

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+* Adjust invoice badge styling (#1906) @bolatovumar
+* Invoice notification email improvements (#1875) @dennisreimann
+* Improvement of the UX flow for requesting an API Key of a BTCPay Server user (#1898) @dennisreimann
+* Don't send notification email for expired invoices @dennisreimann
+* Greenfield API: Add `Roles` property to the user data. @dennisreimann
+* Remove Changelly integration @Kukks
+* Better wording in transaction list page #1887 @maltokyo
+* Fix alerts text break. #1865 @bolatovumar
+* Remove Tor link from navbar @NicolasDorier
+* Improve invoices list view #1815 @dennisreimann
```
Comment (Kukks) [inline Changelog.md:30]:
```suggestion
* Improve invoices list view #1815 @dennisreimann
* Improve sync progress dialog #1929 @Kukks 
```

**Pull request merged**